### DISCUSSION TITLE
motion reported issue and can't capture image

### DISCUSSION THREAD

**Discussion by roseyunzi** [Motion-Project/motion#1712] [Q&A]
I'm using a h3 development board and a usb camera, and here are the logs. Would anyone tell me what's the problem, thanks!

![8eeb3ca3b3417e83edc2b6f011c6b00](https://github.com/Motion-Project/motion/assets/76989423/f74dee99-1ea8-492f-92e0-ffd496454c42)
![34743fce3deb561bac2e043b77f0280](https://github.com/Motion-Project/motion/assets/76989423/29c1e015-c048-4e39-b227-2fae5bd30c8d)
![95ddcb7a8d18c72be887481fe33e5c0](https://github.com/Motion-Project/motion/assets/76989423/e72e5ed1-6c0e-426f-9fe5-3350c1fe4727)

**Comment by tosiara** [comment]
Try updating to the latest 4.5 version first

**Reply by roseyunzi** [reply]
I tried to update but failed.I used apt-get commend to update motion, it said it's already the newest version.
So i switched to another development board, motion running sucessfully for days until this morning, could you tell what's wrong? thank you very much!
![c6444e4259a08d27f1a4744ad8e08467](https://github.com/Motion-Project/motion/assets/76989423/05b3a835-482e-49e5-821d-9329bcb4245f)

**Reply by tosiara** [reply]
Uninstall old motion and install it from github Release
